src/core/public/index.ts

```diff
@@ -143,6 +143,7 @@ export {
 export {
   HttpHeadersInit,
   HttpRequestInit,
+  HttpFetchError,
```

Comment:
Are we OK with this? It's being requested from tests, so either we can add `src/core/public/http` as an exception to the ESLint rule, or we can exempt tests from requiring top level imports.

Reply:
cc: @elastic/kibana-platform